glom/control_flow.py

```diff
@@ -0,0 +1,54 @@
+"""
```

Comment:
I'm thinking that for now this might better fit in matching.py. It may have potential to be more, but pattern matching is what we imagined it for and the `raise GlomError("no matches for target in Switch")` I think really says the whole story.